vector/src/main/java/im/vector/activity/VectorRoomInviteMembersActivity.java

```diff
@@ -38,7 +38,9 @@
 import android.widget.Toast;
 
 import org.matrix.androidsdk.MXSession;
+import org.matrix.androidsdk.data.Room;
```

Comment:
What is the purpose of these changes in VectorRoomInviteMembersActivity class?

Reply:
It is easier to use the Object "Room" to access both roomId and boolean isInvited, while with the roomId have to go get the room and then know if the room isInvited. We must import the class Room of the sdk.